### PR TITLE
Add support for variables

### PR DESCRIPTION
**Pull request by kaicataldo**

### CODE REVIEW

File: test/unit/index.js [hunk 1]
```diff
@@ -24,6 +24,20 @@ define(['index'], function(Focss) {
...
+      it('inserts variables', function() {
+        var variables = {
+          maxHeight: 40,
+          defaultColor: 'red'
+        };
+
+        spyOn(fox.engine, 'insertVars');
+        fox.insertVars(variables);
+        expect(fox.engine.insertVars)
+        .toHaveBeenCalledWith(jasmine.objectContaining(variables));
```
Comment (Attamusc) [inline test/unit/index.js]: Why not just check `fox.variables`?

Comment (kaicataldo) [inline test/unit/index.js]: Could do that - was modeling it after the previous test

Comment (Attamusc) [inline test/unit/index.js]: Ah, then it's fine how it is. Better to be consistent

**Comment by Attamusc** [comment]
👍

**Comment by weikinhuang** [comment]
:+1:

**Pull request merged**